internal/config/settings.go

```diff
@@ -19,6 +19,7 @@ type Settings struct {
 	UsersAPIGRPCAddr           string          `yaml:"USERS_API_GRPC_ADDR"`
 	AccountsAPIGRPCAddr        string          `yaml:"ACCOUNTS_API_GRPC_ADDR"`
 	FetchAPIGRPCEndpoint       string          `yaml:"FETCH_API_GRPC_ENDPOINT"`
+	AttestationAPIGRPCEndpoint string          `yaml:"ATTESTATION_API_GRPC_ENDPOINT"`
```

Comment:
```suggestion
	AttestationAPIGRPCEndpoint string          `yaml:"ATTESTATION_API_GRPC_ADDR"`
```
fix: looks like the var name was updated in the values.yaml/ values-prod.yaml but not here (or vice versa), should update to be consistent